app/Platform/Controllers/SystemController.php

```diff
@@ -34,6 +44,42 @@ public function store(Request $request): void
     {
     }
 
+    public function games(GameListRequest $request, System $system): InertiaResponse
+    {
+        $this->authorize('view', $system);
+
+        /** @var ?User $user */
+        $user = $request->user();
+
+        $isMobile = (new Agent())->isMobile();
+
+        $paginatedData = (new BuildGameListAction())->execute(
+            GameListType::System,
+            targetSystemId: $system->id,
+            user: $user,
+            filters: $request->getFilters(targetSystemId: $system->id),
```

Comment:
These filters don't seem to work. If I change them, there's like a 7 second delay, then the grid clears out:
![image](https://github.com/user-attachments/assets/10556d89-cda2-4d58-a063-f5c7ace993b9)

It still works on the want to play page. I have tried rebooting my VM.

Reply:
Definitely a bug here, and the test was validating that the bug was present. Fixed the code & the test in latest.